### PR TITLE
Log4j update

### PR DESCRIPTION
**Pull request by tharindu1st**
Log4j update

### CODE REVIEW

File: components/application-mgt/org.wso2.carbon.application.upload/pom.xml [hunk 1]
```diff
@@ -70,9 +70,9 @@
             <artifactId>org.wso2.carbon.core</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.wso2.carbon</groupId>
-            <artifactId>org.wso2.carbon.logging</artifactId>
-        </dependency>
+  <groupId>org.ops4j.pax.logging</groupId>
```
Comment (jsdjayanga) [inline components/application-mgt/org.wso2.carbon.application.upload/pom.xml]: Alignment issue

File: components/caching-invalidator/org.wso2.carbon.caching.invalidator/pom.xml [hunk 2]
```diff
@@ -34,9 +34,9 @@
             <artifactId>javax.cache.wso2</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.wso2.carbon</groupId>
-            <artifactId>org.wso2.carbon.logging</artifactId>
-        </dependency>
+  <groupId>org.ops4j.pax.logging</groupId>
```
Comment (jsdjayanga) [inline components/caching-invalidator/org.wso2.carbon.caching.invalidator/pom.xml]: Alignment issue

File: components/event/org.wso2.carbon.event.admin/pom.xml [hunk 3]
```diff
@@ -31,9 +31,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>org.wso2.carbon</groupId>
-            <artifactId>org.wso2.carbon.logging</artifactId>
-        </dependency>
+  <groupId>org.ops4j.pax.logging</groupId>
```
Comment (jsdjayanga) [inline components/event/org.wso2.carbon.event.admin/pom.xml]: Alignment issue

File: components/event/org.wso2.carbon.event.client/pom.xml [hunk 4]
```diff
@@ -32,9 +32,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>org.wso2.carbon</groupId>
-            <artifactId>org.wso2.carbon.logging</artifactId>
-        </dependency>
+  <groupId>org.ops4j.pax.logging</groupId>
```
Comment (jsdjayanga) [inline components/event/org.wso2.carbon.event.client/pom.xml]: Alignment issue

File: components/event/org.wso2.carbon.event.common/pom.xml [hunk 5]
```diff
@@ -32,9 +32,9 @@
 
     <dependencies>
   		<dependency>
-            <groupId>org.wso2.carbon</groupId>
-            <artifactId>org.wso2.carbon.logging</artifactId>
-        </dependency>
+  <groupId>org.ops4j.pax.logging</groupId>
```
Comment (jsdjayanga) [inline components/event/org.wso2.carbon.event.common/pom.xml]: Alignment issue

File: components/ganalytics/pom.xml [hunk 6]
```diff
@@ -36,10 +36,10 @@
             <groupId>org.wso2.orbit.org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
         </dependency>
-        <dependency>
-            <groupId>commons-logging</groupId>
-            <artifactId>commons-logging</artifactId>
-        </dependency>
+      <dependency>
```
Comment (jsdjayanga) [inline components/ganalytics/pom.xml]: Alignment issue

File: components/logging/org.wso2.carbon.logging.updater/pom.xml [hunk 7]
```diff
@@ -14,41 +14,40 @@
...
-	<relativePath>../pom.xml</relativePath>
+        <relativePath>../pom.xml</relativePath>
     </parent>
 
     <modelVersion>4.0.0</modelVersion>
-    <artifactId>org.wso2.carbon.logging.view.ui</artifactId>
+    <artifactId>org.wso2.carbon.logging.updater</artifactId>
     <packaging>bundle</packaging>
-    <name>WSO2 Carbon - Logging View UI</name>
+    <name>WSO2 Carbon - logging config updater</name>
```
Comment (jsdjayanga) [inline components/logging/org.wso2.carbon.logging.updater/pom.xml]: Please use camel case naming

File: components/logging/org.wso2.carbon.logging.updater/pom.xml [hunk 8]
```diff
@@ -14,41 +14,40 @@
...
+                        <Import-Package>
+                            !org.wso2.carbon.logging.updater.*,
+                            org.wso2.carbon.utils.*;version="${carbon.kernel.imp.pkg.version}",
+                            org.osgi.service.cm.*,
                             *;resolution:=optional
-			</Import-Package>
-			<!--Embed-Dependency>org.wso2.carbon.logging.view.stub;scope=compile|runtime;inline=true</Embed-Dependency-->
-                        <Carbon-Component>UIBundle</Carbon-Component>
+                        </Import-Package>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```
Comment (jsdjayanga) [inline components/logging/org.wso2.carbon.logging.updater/pom.xml]: Can we remove this if there is no requirement?

File: components/logging/org.wso2.carbon.logging.updater/pom.xml (hunk 7 above)
Comment (madushadhanushka) [reply components/logging/org.wso2.carbon.logging.updater/pom.xml]: https://github.com/wso2/carbon-commons/pull/369

File: components/logging/org.wso2.carbon.logging.updater/pom.xml (hunk 8 above)
Comment (madushadhanushka) [reply components/logging/org.wso2.carbon.logging.updater/pom.xml]: Fixed with https://github.com/wso2/carbon-commons/pull/369

File: components/ganalytics/pom.xml (hunk 6 above)
Comment (madushadhanushka) [reply components/ganalytics/pom.xml]: Fixed with https://github.com/wso2/carbon-commons/pull/369

**Pull request merged**